app/services/green_lanes/eligibility_result.rb

```diff
@@ -0,0 +1,58 @@
+module GreenLanes
+  class EligibilityResult
+    YOU_MAY_BE_ELIGIBLE = 'you_may_be_eligible'.freeze
+    NOT_YET_ELIGIBLE = 'not_yet_eligible'.freeze
+    NOT_ELIGIBLE = 'not_eligible'.freeze
+
+    NOT_SURE = 'not_sure'.freeze
+    YES = 'yes'.freeze
```

Comment:
is `NO` missing?

Reply:
actually its not used so dont need this here to be honest..